web/src/components/app-survey/FeedbackCard.tsx

```diff
@@ -1,29 +1,80 @@
 import * as ToggleGroupPrimitive from '@radix-ui/react-toggle-group';
 import Pill from 'components/Pill';
-import { ChangeEvent, Dispatch, SetStateAction, useState } from 'react';
+import { useAtom, useSetAtom } from 'jotai';
+import {
+  ChangeEvent,
+  Dispatch,
+  SetStateAction,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
 import { useTranslation } from 'react-i18next';
 import { HiOutlineX } from 'react-icons/hi';
+import { hasSeenSurveyCardAtom, userLocationAtom } from 'utils/state/atoms';
 
 enum FeedbackState {
   INITIAL = 'initial',
   OPTIONAL = 'optional',
   SUCCESS = 'success',
 }
 
+export interface SurveyResponseProps {
+  feedbackScore: number;
+  inputText: string;
+  reference: string;
+  location?: string;
+}
+interface FeedbackCardProps {
+  postSurveyResponse: (props: SurveyResponseProps) => void;
+  subtitle?: string;
+  primaryQuestion: string;
+  secondaryQuestionHigh?: string;
+  secondaryQuestionLow?: string;
+  surveyReference?: string;
+}
+
 export default function FeedbackCard({
-  estimationMethod,
-}: {
-  estimationMethod?: string;
-}) {
+  postSurveyResponse,
+  subtitle,
+  primaryQuestion,
+  secondaryQuestionHigh,
+  secondaryQuestionLow,
+
+  surveyReference,
+}: FeedbackCardProps) {
   const [isClosed, setIsClosed] = useState(false);
   const [feedbackState, setFeedbackState] = useState(FeedbackState.INITIAL);
+  const setHasSeenSurveyCard = useSetAtom(hasSeenSurveyCardAtom);
 
   const handleClose = () => {
     setIsClosed(true);
+    setHasSeenSurveyCard(true);
   };
```

Comment:
This has the unintended side-effect that closing the "estimations feedback" box also closes this new survey 😅 


https://github.com/electricitymaps/electricitymaps-contrib/assets/3296643/060ed327-47b0-4631-bec9-e0e812d9a5a1

Reply:
Should be fixed now!